### ISSUE TITLE
Popper 2.x TODO

### ISSUE THREAD

**Issue by XhmikosR** [twbs/bootstrap#32354]
- [x] ~~Update Popper docs links to v2 (search for `https://popper.js.org/docs/` but don't blindly replace them!)~~ Fixed by #32390
- [x] ~~Update docs after #32405~~ Fixed by #32437 (Only for Tooltip/Popover)
- [x] ~~Update the docs for `fallbackPlacements` for dropdown — https://github.com/twbs/bootstrap/issues/32506#issuecomment-746509273~~ Changes were reverted in #32843 
- [ ] Add more JS tests to cover the changes in #31178

**Comment by septatrix** [comment]
#32365 should probably also be a part of this

**Comment by XhmikosR** [comment]
Not really, unless a PR is submitted.

**Comment by septatrix** [comment]
Okay, I will submit one if that is okay.

I also noticed that popper shows a warning in the console (at least when referencing the scripts manually and not using the minified bundle). Look inside the console of [this pen](https://codepen.io/Septatrix/pen/dypXKgx?editors=1111) when clicking the button. This is due to `.dropdown-menu` specifying a margin which poppler overwrites anyway. Do you want to track this here or create a new issue?

**Comment by XhmikosR** [comment]
Please make a new issue about the second thing. We should find a way to fix this before the stable release :)

**Comment by XhmikosR** [comment]
@rohit2sharma95 do we still need to cover the last TODO here?

**Comment by rohit2sharma95** [comment]
Yes, I see there are some missing tests. I will update the draft PR.
